python/hsfs/engine/python.py

```diff
@@ -1305,3 +1301,15 @@ def is_connector_type_supported(connector_type):
             StorageConnector.SNOWFLAKE,
             StorageConnector.KAFKA,
         ]
+
+    @staticmethod
+    def _start_offline_materialization(offline_write_options):
+        if offline_write_options is not None:
+            if "start_offline_materialization" in offline_write_options:
+                return offline_write_options.get("start_offline_materialization")
+            elif "start_offline_backfill" in offline_write_options:
+                return offline_write_options.get("start_offline_backfill")
+            else:
+                return True
+        else:
+            return False
```

Comment:
Shouldn't this be returning True? if there are no options, the default behavior is to start the materialization job.